src/Sylius/Bundle/ShopBundle/Twig/Component/Product/CountReviewsComponent.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ShopBundle\Twig\Component\Product;
+
+use Sylius\Component\Channel\Context\ChannelContextInterface;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Core\Model\Product;
+use Sylius\Component\Core\Model\ProductReview;
+use Sylius\Component\Core\Repository\ProductReviewRepositoryInterface;
+use Sylius\Component\Locale\Context\LocaleContextInterface;
+use Symfony\UX\TwigComponent\Attribute\AsTwigComponent;
+use Symfony\UX\TwigComponent\Attribute\ExposeInTemplate;
+
+#[AsTwigComponent]
+class CountReviewsComponent
+{
+    public Product $product;
+
+    /**
+     * @param ProductReviewRepositoryInterface<ProductReview> $productReviewRepository
+     */
+    public function __construct(
+        private readonly ProductReviewRepositoryInterface $productReviewRepository,
+        private readonly ChannelContextInterface $channelContext,
+        private readonly LocaleContextInterface $localeContext,
+    ) {
+    }
+
+    #[ExposeInTemplate('reviews_count')]
+    public function countReviews(): int
+    {
+        /** @var ChannelInterface $channel */
+        $channel = $this->channelContext->getChannel();
+
+        return count($this->productReviewRepository->findAcceptedByProductSlugAndChannel($this->product->getSlug(), $this->localeContext->getLocaleCode(), $channel));
```

Comment:
IMO, create a new method for that:
```suggestion
        return count($this->productReviewRepository->findAcceptedByProduct($this->product));
```
or even better:
```suggestion
        return $this->productReviewRepository->countAcceptedByProduct($this->product);
```